spec/middleman-prismic/preview_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "spec_helper"
+require "middleman-prismic/preview"
+
+describe Middleman::Prismic::Preview do
+  it "passes through requests outside of /preview" do
+    app = rack_app
+    allow(Kernel).to receive(:system)
```

Comment:
What do you think about putting this one line in a `before` block? I don't feel like it adds much to the readability of the test, and it's repeated in all examples.

Reply:
good idea. 33c95f9